control/controlclient/noise.go

```diff
@@ -53,6 +54,10 @@ type noiseClient struct {
 	httpPort     string // the default port to call
 	httpsPort    string // the fallback Noise-over-https port
 
+	// Optional DialPlan received from the control server previously; can
```

Comment:
If it's a before-line comment, use a complete sentence.

// dialPlan optionally ...